OpenMEEG/libs/OpenMEEG/include/gain.h

```diff
@@ -53,6 +53,7 @@ namespace OpenMEEG {
     class GainMEG : public Matrix {
     public:
         using Matrix::operator=;
+        GainMEG (const Matrix& GainMat): Matrix(GainMat) {}
         GainMEG (const SymMatrix& HeadMatInv,const Matrix& SourceMat, const Matrix& Head2MEGMat, const Matrix& Source2MEGMat) {
             *this = Source2MEGMat+(Head2MEGMat*HeadMatInv)*SourceMat;
```

Comment:
yes it was for the python bindings ... So it is still possible to use it for python. 
But if the user is short on memory he should do himself the gain operations releasing each matrix ASAP when doing the products....
I don't see another way for doing that... or maybe with filenames instead of matrices for the constructor.... ?